image_lists/rrfs_subset.yml

```diff
@@ -92,7 +94,7 @@ hourly:
       - sfc
     ptmp:
       - 2m
-    ptyp
+    ptyp:
```

Comment:
I noticed this typo while adding firewx to the file.

Reply:
I guess that was my fault.  Sorry.  Thanks for fixing it.